packages/client/css/glsp-sprotty.css

```diff
@@ -73,11 +73,6 @@
     fill: #1d80d1;
 }
 
```

Comment:
Quick question: How do we now make sure that resize handles are not considered for bounds computation?

Reply:
And a very good question at that! Unfortunately, I simply couldn't reproduce the original issue anymore. It seems that the `getBBox` simply returns the correct bounds now on all browsers for all elements.